samples/batch/transcription-enabled-storage/Connector/AudioFileProcessor.cs

```diff
@@ -0,0 +1,106 @@
+﻿// <copyright file="AudioFileProcessor.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System;
+    using System.IO;
+    using NAudio.Wave;
+
+    public static class AudioFileProcessor
+    {
+        // prices in euro
+        private const double STTCostPerHour = 0.844d;
+
+        private const double STTCustomModelCostPerHour = 1.181d;
+
+        private const double TextAnalyticsCostPerHour = 0.72d;
+
+        public static TimeSpan GetDuration(byte[] fileBytes, string fileNameExtension)
```

Comment:
This only works for WAV. Please add a comment to make it clear.

Reply:
as discussed: removed the receipt file, therefore this is no longer needed